app/src/components/project/Integrations.tsx

```diff
@@ -0,0 +1,280 @@
+    githubHref:
+      "https://github.com/Arize-ai/openinference/tree/main/python/instrumentation/openinference-instrumentation-groq",
+    icon: <GroqSVG />,
+  },
+  {
+    name: "CrewAI",
+    docsHref:
+      "https://docs.arize.com/phoenix/tracing/integrations-tracing/crewai",
+    githubHref:
+      "https://github.com/Arize-ai/openinference/tree/main/python/instrumentation/openinference-instrumentation-crewai",
+    icon: <CrewAISVG />,
+  },
+];
+
+const integrationsListCSS = css`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  gap: var(--ac-global-dimension-size-100);
+  flex-wrap: wrap;
+`;
+
+export function PythonIntegrations() {
+  return (
+    <ul css={integrationsListCSS}>
+      {PYTHON_INTEGRATIONS.map((integration) => (
+        <li key={integration.name}>
+          <IntegrationLink key={integration.name} {...integration} />
+        </li>
+      ))}
+    </ul>
+  );
+}
+
+const TYPESCRIPT_PLATFORM_INTEGRATIONS: IntegrationLinkProps[] = [
+  {
+    name: "Node",
+    docsHref:
+      "https://opentelemetry.io/docs/languages/js/getting-started/nodejs/",
+    githubHref: "https://github.com/open-telemetry/opentelemetry-js",
+    icon: <NodeJSSVG />,
+  },
+  {
+    name: "Vercel",
+    docsHref: "https://vercel.com/docs/observability/otel-overview",
+    githubHref: "https://github.com/vercel/otel",
+    icon: <VercelSVG />,
+  },
+];
+export function TypeScriptPlatformIntegrations() {
+  return (
+    <ul css={integrationsListCSS}>
+      {TYPESCRIPT_PLATFORM_INTEGRATIONS.map((integration) => (
+        <li key={integration.name}>
+          <IntegrationLink key={integration.name} {...integration} />
+        </li>
+      ))}
+    </ul>
+  );
+}
+
+const TYPESCRIPT_INTEGRATIONS: IntegrationLinkProps[] = [
+  {
+    name: "OpenAI NodeJS SDK",
+    docsHref:
+      "https://docs.arize.com/phoenix/tracing/integrations-tracing/openai-node-sdk",
+    githubHref:
+      "https://github.com/Arize-ai/openinference/tree/main/js/packages/openinference-vercel",
+    icon: <OpenAISVG />,
+  },
+  {
+    name: "LangChain.js",
+    docsHref:
+      "https://docs.arize.com/phoenix/tracing/integrations-tracing/langchain.js",
+    githubHref:
+      "https://github.com/Arize-ai/openinference/tree/main/js/packages/openinference-instrumentation-langchain",
+    icon: <LangChainSVG />,
+  },
+  {
+    name: "Vercel AI SDK",
+    docsHref:
+      "https://docs.arize.com/phoenix/tracing/integrations-tracing/vercel-ai-sdk",
+    githubHref:
+      "https://github.com/Arize-ai/openinference/tree/main/js/packages/openinference-vercel",
+    icon: <VercelSVG />,
+  },
+];
+export function TypeScriptIntegrations() {
+  return (
+    <ul css={integrationsListCSS}>
+      {TYPESCRIPT_INTEGRATIONS.map((integration) => (
+        <li key={integration.name}>
+          <IntegrationLink key={integration.name} {...integration} />
+        </li>
+      ))}
+    </ul>
+  );
+}
+
+const GitHubSVG = () => (
```

Comment:
nit: move this to it's own file

Reply:
I'm going a bit rogue here because I don't want to re-use this quite yet.